.lgtm.yml

```diff
@@ -0,0 +1,40 @@
+# See https://lgtm.com/
+extraction:
+  cpp:
+    prepare:
+      packages:
+      - "autoconf-archive"
```

Comment:
I cargo culted these from `travis-install.sh` - obviously annoying to duplicate, at least since this one is declarative I can see about parsing it for our Debian/Ubuntu builds.